### PR TITLE
265 ic footer to use ic link

### PR DESCRIPTION
**Pull request by GCHQ-Developer-299**
## Summary of the changes
Tell us, in a few words, what the changes are.

## Related issue
Tell us the issue number. If suggesting an improvement or component, please discuss it with us in an issue first.

### General 

- [x] Changes to docs package checked and committed.
- [x] All acceptance criteria reviewed and met. 

### Testing

- [x] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 
- [x] Playground stories in React Storybook up to date, with any prop changes and additions addressed. 
- [x] Compare performance of modified components against develop using Performance addon in React Storybook.

### Accessibility 

- [x] Accessibility Insights FastPass performed.
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 

### Resize/zoom behaviour 

- [x] Page can be zoomed to 400% with no loss of content. 
- [x] Screen magnifier used with no issues. 
- [x] Text resized to 200% with no loss of content.
- [x] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.

### System modes

- [x] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [x] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content.
- [x] Browser support tested (Chrome, Safari, Firefox and Edge). 

### Testing content extremes

- [x] Min/max content examples tested with no loss of content or overflow. 
- [x] All prop combinations work without issue. 
- [x] Tested for FOUC (Flash of Unstyled Content) in both SSR (Server-Side Rendering) and SSG (Static Site Generation) settings.
- [x] Controlled and uncontrolled input components tested.
- [x] Props/slots can be updated after initial render.

### CODE REVIEW

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by GCHQ-Developer-530 — DISMISSED** [review]

File: packages/web-components/src/components/ic-footer-link/ic-footer-link.css [hunk 1]
```diff
@@ -30,140 +30,118 @@
...
-:host(.footer-link) a:link:hover,
-:host(.footer-link) a ::slotted(a:link:hover) {
-  outline: none;
+:host(.link) .ic-link:active,
+:host(.link) .ic-link:focus:active,
+:host(.link) .ic-link:visited:active,
+:host(.link) ::slotted(a:active),
+:host(.link) ::slotted(a:focus:active),
+:host(.link) ::slotted(a:visited:active) {
+  text-decoration: none;
```
Comment (jd239) [inline packages/web-components/src/components/ic-footer-link/ic-footer-link.css:85]: The CSS classes have been changed to `.link` but the host still has `.footer-link`. Is there supposed to be a change in host CSS classes?

Comment (jd239) [reply packages/web-components/src/components/ic-footer-link/ic-footer-link.css:85]: I can see there is a `.link` class on `ic-link` if it helps...

**Review by jd239 — COMMENTED** [review]
Just a single comment

File: packages/web-components/src/components/ic-footer-link/ic-footer-link.css (hunk 1 above)
Comment (GCHQ-Developer-299) [reply packages/web-components/src/components/ic-footer-link/ic-footer-link.css]: Looks like this CSS was redundant anyways - as you say, the link class on ic-link handles all of this :)

**Review by jd239 — APPROVED** [review]

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Pull request merged**